suite-common/wallet-core/src/token-definitions/tokenDefinitionsSelectors.ts

```diff
@@ -12,3 +14,16 @@ export const selectSpecificTokenDefinition = (
     networkSymbol: NetworkSymbol,
     contractAddress: string,
 ) => state.wallet.tokenDefinitions?.[networkSymbol]?.[contractAddress];
+
+export const selectKnownNetworkTokens = (
+    state: TokenDefinitionsRootState,
+    networkSymbol: NetworkSymbol,
+) => {
+    const networkTokenDefinitions = selectTokenDefinitions(state, networkSymbol);
+
+    return pipe(
```

Comment:
wouldnt this trigger rebuilding since we are returning new array every time?

Maybe not in this usage but if used later somewhere else?

Reply:
I think this selector is not so computational expesive so memoization is not needed here. Also as you said in the case where it is used, there is no re-render issue. So I would leave it as it is. It can be momemoized later if needed elsewhere. I believe it is better to not optimize memoization prematurely.